scheds/kernel-examples/scx_userland.h

```diff
@@ -4,6 +4,9 @@
 #ifndef __SCX_USERLAND_COMMON_H
 #define __SCX_USERLAND_COMMON_H
 
+/*
+ * Maximum amount of tasks enqueued/dispatched between kernel and user-space.
+ */
 #define USERLAND_MAX_TASKS 8192
```

Comment:
Is this macro still used? If not let's just delete it?

Reply:
It's still used in .bpf.c. There, it's just the max size of the ring buffer and overflow handling seems okay. Maybe it should be renamed for clarity?

Reply:
Gah, right, my bad. Yeah, maybe something like `MAX_ENQUEUED_TASKS`? And we can probably just bring it into scx_userland.bpf.c directly. It's also an option to set the max entries value from userspace before the map is loaded, but this seems preferable given the fallback logic as you pointed out.

Reply:
It's quite an internal parameter, basically it defines how many items can be pushed at most to the enqueued/dispatched maps, so I don't think there's much value to export this to user-space as a configurable option... but I think we can definitely bring it into .bpf.c, since it's used only there at this point. And I like the name `MAX_ENQUEUED_TASKS`, it's definitely more clear.

Reply:
Oh yeah, I meant that you could set the max entries in the map to the value of `/proc/sys/kernel/pid_max`. But yeah I think bringing the macro into the bpf file and statically setting it there is fine / preferable.

Reply:
... and updated.

Setting pid-max entries in the bpf maps would be a bit of a waste IMHO, there's no way a system will have pid-max concurrent tasks running. :)

And if we fail to enqueue a task from in the bpf part the task will just be dispatched immediately. OTOH if we fail to push the task to the dispatched map from the .c we would error and exit, maybe this part can be improved a bit, i.e., re-adding the pid to the vruntime_head list?

Reply:
Agreed, sounds like another nice improvement!